sdk/messaging/azservicebus/internal/amqp_test_utils.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
If these artifacts are used solely for tests you should rename the file to amqp_utils_test.go so they're only available in tests.

Reply:
I think I'm getting bit again by this whole separate-package bit again. 

These are also used in the main package's tests and the _test files in `internal` don't get included in that case.

Reply:
Ah yes good point.